README.md

```diff
@@ -100,6 +100,7 @@ For additional help, join us [in our Slack](https://now-examples-slackin-rrirkqo
 -   [storyshots](addons/storyshots/) - Easy snapshot testing for components in Storybook
 -   [storysource](addons/storysource/) - View the code of your stories within the Storybook UI
 -   [viewport](addons/viewport/) - Change display sizes and layouts for responsive components using Storybook
+-   [styled-component-theme](https://github.com/echoulen/storybook-addon-styled-component-theme) - Styled-components theme selection.
```

Comment:
This list is only for the addons from the monorerpo.  
Please add this to the https://storybook.js.org/addons/addon-gallery/ (`docs/src/pages/addons/addon-gallery/index.md`)

Reply:
got it